config/initializers/assets.rb

```diff
@@ -14,4 +14,4 @@
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
 Rails.application.config.assets.precompile += %w( react-app.js stats.js )
 Rails.application.config.assets.precompile += %w( favicon.ico )
-Rails.application.config.assets.precompile += %w( *.woff *.woff2 *.eot *.ttf )
+Rails.application.config.assets.precompile += %w( *.woff *.woff2 *.eot *.ttf )
```

Comment:
Looks like we're missing another newline here

```suggestion
Rails.application.config.assets.precompile += %w( *.woff *.woff2 *.eot *.ttf )

```